internal/common/labels.go

```diff
@@ -18,6 +18,7 @@ const (
 	AppComponentTektonTasks           AppComponent = "tektonTasks"
 	AppKubernetesManagedByValue       string       = "ssp-operator"
 	TektonAppKubernetesManagedByValue string       = "tekton-tasks-operator"
+	AppKubernetesPartOfValue          string       = "hyperconverged-cluster"
```

Comment:
Is that always the case?

Reply:
This value should not be hardcoded. It is copied from the SSP CR:
https://github.com/kubevirt/ssp-operator/blob/e92bc396eca2195d7d43b30d8f9fb363fb418f47/internal/common/labels.go#L36-L49

Because it is set by HCO or OLM.